RoddyCore/src/de/dkfz/roddy/Roddy.java

```diff
@@ -593,12 +593,26 @@ private static void setDefaultRoddyQueueVariable(RecursiveOverridableMapContaine
         configurationValues.add(new ConfigurationValue(CVALUE_PLACEHOLDER_RODDY_QUEUE_RAW, "$" + Roddy.jobManager.getJobIdVariable()));
     }
 
-    /** Take RODDY_SCRATCH, or if this is empty "defaultScratchDir" as scratch base directory. The job-manager specific _JOBID variable (e.g.
-     *  PBS_JOBID) is then appended as "/${PBS_JOBID}", to ensure that every job gets its own scratch directory.
+    /** Copy the scratchBaseDirectory value from the applicationProperties.ini into the configuration. Set the default value for RODDY_SCRATCH
+     *  to $scratchBaseDirectory/$RODDY_JOBID.
      * @return
      */
-    private static void setDefaultRoddyScratchVariable(RecursiveOverridableMapContainerForConfigurationValues configurationValues) {
-        String scratchDir = new File(ConfigurationConstants.DEFAULT_SCRATCH_DIR, "$" + Roddy.jobManager.getJobIdVariable()).getAbsolutePath();
+    private static void setScratchDirectory(RecursiveOverridableMapContainerForConfigurationValues configurationValues) {
+        String scratchBaseDir = Roddy.getApplicationProperty(Constants.APP_SCRATCH_BASE_DIRECTORY, "");
+        if (scratchBaseDir.equals("")) {
+            File propertyFilePath = null;
+            try {
```

Comment:
Do you need the check at all? Either the file exists and Roddy will start or it doesn't and Roddy will stop. How about default ini files?